src/components/fields/select-users.js

```diff
@@ -36,22 +37,27 @@ class SelectUsers extends React.Component {
 					loading: true
 				} );
 
-				// Query for the users.
-				return Observable.ajax( {
-					url: 'https://meta.wikimedia.org/w/api.php?action=query&format=json&list=globalallusers&origin=*&agufrom=' + encodeURIComponent( this.ucFirst( input ) ),
-					crossDomain: true
-				} )
-					.map( ( ajaxResponse ) => {
-						return ajaxResponse.response.query.globalallusers.map( ( user ) => {
-							return {
-								label: user.name,
-								value: user.name
-							};
-						} );
+				if ( isIPAddress( input ) ) {
```

Comment:
These changes could use some comments to explain what exactly is happening here.

Reply:
I thought it was self explanatory but I can add a line if you think it is necessary